hack/release/packaging/utils/create-update-packages.sh

```diff
@@ -82,22 +95,28 @@ function require_repo_name {
 function require_rpm_host_vars {
     # HOST and GCLOUD_ARGS must be set to indicate the RPM host, and a
     # gcloud identity that permits logging into that host.
-    test -n "$GCLOUD_ARGS"
-    echo GCLOUD_ARGS is "$GCLOUD_ARGS"
-    test -n "$HOST"
-    echo HOST is $HOST
+    if [[ -z  "$GCLOUD_ARGS" ]]; then
```

Comment:
nit: I believe you could just use single brackets here.  (+ same just below)

Reply:
That's true in this case, but I'm in the habit of using `[[ .. ]]` instead of `[ .. ]` because of the various syntax improvements it provides and the fewer gotchas that can happen later on.

(http://mywiki.wooledge.org/BashFAQ/031 for reference if anyone is curious)